### ISSUE TITLE
connect web3.js to react

### ISSUE THREAD

**Issue by gorillya** [kompilr/dao#2]

**Comment by malnakli** [comment]
@gorillya I think is good to use this from the start https://webpack.js.org/

**Comment by gorillya** [comment]
@malnakli i agree about webpack in create-react-app, but how exactly does the web3 fit within a component of app.js only has the root div.. I dont understand yet. If you have a link send it to me.

**Comment by malnakli** [comment]
@gorillya  check this  https://github.com/uzyn/ethereum-webpack-example-dapp, it could help
